overview.md

```diff
@@ -43,130 +43,173 @@ steps:
 
-Argument                  | Description    
-------------------------- | --------------- 
-`release`                 | (Optional) MATLAB release to install. You can specify R2021a or a later release. By default, the value of `release` is `latest`. If you do not specify `release`, the task installs the latest release of MATLAB.<br/>**Example:** `release: R2023b`<br/>**Example:** `release: latest`
-`products`                | (Optional) Products to install in addition to MATLAB, specified as a list of product names separated by spaces. You can specify `products` to install most MathWorks&reg; products and support packages. For example, `products: Deep_Learning_Toolbox` sets up Deep Learning Toolbox&trade; in addition to MATLAB. The task uses [MATLAB Package Manager](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md) (`mpm`) to install products. For a list of supported products and their correctly formatted names, see [Product Installation Options](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/MPM.md#product-installation-options).<br/>**Example:** `products: Simulink`<br/>`products: Simulink Deep_Learning_Toolbox`
+#### Licensing
+Product licensing for your pipeline depends on your project type as well as the type of products to install:
+
+- Public project — If your pipeline does not include transformation products, such as MATLAB Coder and MATLAB Compiler, then the task automatically licenses any products that you install. If your pipeline includes transformation products, you can request a MATLAB batch licensing [token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token) by contacting MathWorks at [batch-tokens@mathworks.com](mailto:batch-tokens@mathworks.com).
+- Private project — The task does not automatically license any products for you. You can request a batch licensing token by contacting MathWorks.
+  
+To use a MATLAB batch licensing token, first set it as a [secret variable](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/set-secret-variables?view=azure-devops&tabs=yaml%2Cbash). Then, map the secret variable into an environment variable named `MLM_LICENSE_TOKEN` in your YAML pipeline. For an example, see [Use MATLAB Batch Licensing Token](#use-matlab-batch-licensing-token). 
```

Comment:
It's important that the user maps the env variable in the `Run*` step, not the `Install` step. I know the example shows that, but we might want to point it out in the text as well.

Reply:
Updated the item text to the following. What do you think?

Map the secret variable into an environment variable named `MLM_LICENSE_TOKEN` in each of the **Run MATLAB Build**, **Run MATLAB Tests**, and **Run MATLAB Command** tasks of your YAML pipeline.